pac2go.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+function die() {
+  echo $*
+  exit 1
+}
+which 2goarray > /dev/null || go get github.com/jteeuwen/go-bindata/...
+which 2goarray > /dev/null || die "Please install 2goarray manually, then try again"
```

Comment:
I tried go-bindata, but it requires the file to be under a folder, so I have to create seperate folder for each OS. 2goarray doesn't even impose runtime dependency, which is very good.